platform/server/plugins/PluginsService.ts

```diff
@@ -78,23 +78,29 @@ export class PluginsService implements CoreService {
       throw new Error(`'plugin' definition missing in plugin [${pluginPath}]`);
     }
 
-    if (!('dependencies' in json)) {
+    const { plugin } = json;
```

Comment:
__question:__ why is it called `json` instead of plugin exports or something like this?

Reply:
Because reasons. Definitely. I'm sure of that.

Also: ✅ 

🙈

Reply:
:D 🥇